tutorial/westeros/westeros_flexible_generation.ipynb

```diff
@@ -68,9 +70,7 @@
    "source": [
     "## Add a carbon tax\n",
     "\n",
-    "Note that the example below is not feasible with a strict bound on emissions, hence this tutorial uses a tax.\n",
-    "\n",
-    "Below, we repeat the set-up of the emissions formulation from the \"emissions bounds\" tutorial."
+    "Then, we add a carbon tax to motivate the use of low-carbon technologies in the system. We do this similar to the process explained in the [tutorial for adding emission tax](https://github.com/iiasa/message_ix/blob/master/tutorial/westeros/westeros_emissions_taxes.ipynb)"
```

Comment:
I would refer to the notebook by its file name here, rather than a hyperlink. That will be sensible even if the user is offline.